src/frontend/Komponenter/Behandling/Inngangsvilkår/Aleneomsorg/AleneomsorgInfo.tsx

```diff
@@ -31,124 +29,101 @@ const AleneomsorgInfo: FC<{
     ) => {
         const harLøpendeStønad = barnMedLøpendeStønad.barn.includes(personIdent);
         return harLøpendeStønad ? (
-            <Tag variant={'success'}>{`ja - per ${formaterNullableIsoDato(
+            <Tag variant={'success'} size={'small'}>{`ja - per ${formaterNullableIsoDato(
                 barnMedLøpendeStønad.dato
             )}`}</Tag>
         ) : (
-            <Tag variant={'error'}>{`nei - per ${formaterNullableIsoDato(
+            <Tag variant={'error'} size={'small'}>{`nei - per ${formaterNullableIsoDato(
                 barnMedLøpendeStønad.dato
             )}`}</Tag>
         );
     };
 
+    const navnOgAlderPåBarn = registergrunnlag.navn
+        ? utledNavnOgAlder(
+              registergrunnlag.navn,
+              registergrunnlag.fødselsdato,
+              registergrunnlag.dødsdato
+          )
+        : søknadsgrunnlag.navn && søknadsgrunnlag.navn !== ''
```

Comment:
Trenger ikke å sjekke om den ikke er tom string :) `''` evalueres til falsy
```suggestion
        : søknadsgrunnlag.navn
```